docs/docs/getting-started/quick-start.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Quick Start
+sidebar_label: Quick Start
+sidebar_position: 2
+---
+
+import CodeBlock from '@theme/CodeBlock'; import { JsVersion } from '@site/src/components/SdkVersions.js';
+
+# Flagsmith Quick Start Guide
+
+This tutorial will guide you through the core concepts of integrating Flagsmith into your application.
+
+## Learning objectives
+
+By the end of this tutorial, you will be able to:
+
+- Create a Project and your first Flag in the Flagsmith dashboard.
```

Comment:
As per other conversations, please refrain from capitalising terms that don't need it. 

```suggestion
- Create a Project and your first flag in the Flagsmith dashboard.
```